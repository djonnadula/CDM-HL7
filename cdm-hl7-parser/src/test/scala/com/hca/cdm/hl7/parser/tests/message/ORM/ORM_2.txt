MSH|^~\&||COCBR|||201705160000||ORM^O01|MT_COCBR_ORM_BROEO.1.17377700|P|2.1|||||
PID|1||H000962893|H890237|LOVEE^ARRIBA||19630426|M||W|30 MURDLER PLOOP^^PLANTUBUS CITY^AK^12345|||(999)999-9999||S|NON|H00728054711|123-23-5555
PV1|1|I|H.A4PCU^H.A401^1|EM|||CALRIA^Calata^Ria^^^^MD^^^^^^XX|.SELF^Referred^Self^^^^^^^^^^XX|KAHRA^Kahn^Randy^M^^^MD^^^^^^XX|MEDTELE||||PR|||CALRIA^Calata^Ria^^^^MD^^^^^^XX|IN||15|||||||||||||||||||COCBR||ADM|||201705152228||||||||.SELF^Referred^Self^^^^^^^^^^XX
ORC|NW|||||N|||201705152359|HSK1||||||||BRHDME1W011B.2
OBR|1|20170515-0365||CONS^PHA^PHARMACY CONSULT||201705152359||||||ACOEISO^Precautions:||||SHISAN^Kagathur^Santosh^PGY1^^^MD|||||||||||^^^^^S
NTE|1|P|BRPHACON^Select Pharmacy Consult^V^Vancomycin Rx to Dose^Pharmacy Consults Brandon
NTE|2|P|HOECAMP^Test to be performed at?^^BR
ZMN|