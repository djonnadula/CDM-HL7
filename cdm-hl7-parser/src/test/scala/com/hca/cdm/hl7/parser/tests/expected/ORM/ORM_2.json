{
  "0000.COMN": {
    "sending_facility": "COCBR",
    "message_control_id": "MT_COCBR_ORM_BROEO.1.17377700",
    "medical_record_num": "H154123134",
    "medical_record_urn": "H890237",
    "patient_account_num": "H00728054711"
  },
  "0001.MSH": {
    "field_separator": "|",
    "encoding_characters": "^~\\&",
    "sending_application": null,
    "sending_facility": "COCBR",
    "receiving_application": null,
    "receiving_facility": null,
    "date_time_of_message": "201705160000",
    "security": null,
    "message_type": {
      "message_code": "ORM",
      "trigger_event": "O01"
    },
    "message_control_id": "MT_COCBR_ORM_BROEO.1.17377700",
    "processing_id": "P",
    "version_id": "2.1"
  },
  "0002.PID": {
    "set_id": "1",
    "lss_encounter_num": null,
    "medical_record_num": "H154123134",
    "medical_record_urn": "H890237",
    "patient_name": {
      "family_name": "LOVEE",
      "given_name": "ARRIBA"
    },
    "mother_maiden_name": null,
    "date_time_of_birth": "19630426",
    "administrative_sex": "M",
    "patient_alias": null,
    "race": "W",
    "patient_address": {
      "street_address": "30 MURDLER PLOOP",
      "other_designation": null,
      "city": "PLANTUBUS CITY",
      "state_or_province": "AK",
      "zip_or_postal_code": "12345"
    },
    "county_code": null,
    "phone_num_home": null,
    "phone_num_business": "(999)999-9999",
    "primary_language": null,
    "marital_status": "S",
    "religion": "NON",
    "patient_account_num": "H00728054711",
    "ssn_num_patient": "999-99-9999"
  },
  "0003.PV1": {
    "set_id": "1",
    "patient_class": "I",
    "assigned_patient_location": {
      "point_of_care": "H.A4PCU",
      "room": "H.A401",
      "bed": "1"
    },
    "admission_type": "EM",
    "pre_admit_num": null,
    "prior_patient_location": null,
    "attending_doctor": {
      "person_id": "CALRIA",
      "family_name": "Calata",
      "given_name": "Ria",
      "second_and_further_given_names_or_initials_thereof": null,
      "suffix": null,
      "prefix": null,
      "degree": "MD",
      "source_table": null,
      "assigning_authority": null,
      "name_type_code": null,
      "id_check_digit": null,
      "check_digit_scheme": null,
      "id_type_code": "XX"
    },
    "referring_doctor": {
      "person_id": ".SELF",
      "family_name": "Referred",
      "given_name": "Self",
      "second_and_further_given_names_or_initials_thereof": null,
      "suffix": null,
      "prefix": null,
      "degree": null,
      "source_table": null,
      "assigning_authority": null,
      "name_type_code": null,
      "id_check_digit": null,
      "check_digit_scheme": null,
      "id_type_code": "XX"
    },
    "primary_care_doctor": {
      "person_id": "KAHRA",
      "family_name": "Kahn",
      "given_name": "Randy",
      "second_and_further_given_names_or_initials_thereof": "M",
      "suffix": null,
      "prefix": null,
      "degree": "MD",
      "source_table": null,
      "assigning_authority": null,
      "name_type_code": null,
      "id_check_digit": null,
      "check_digit_scheme": null,
      "id_type_code": "XX"
    },
    "hospital_service": "MEDTELE",
    "temporary_location": null,
    "pre_admit_test_indicator": null,
    "readmission_indicator": null,
    "admit_source": "PR",
    "mode_of_arrival": null,
    "vip_indicator": null,
    "admitting_doctor": {
      "person_id": "CALRIA",
      "family_name": "Calata",
      "given_name": "Ria",
      "second_and_further_given_names_or_initials_thereof": null,
      "suffix": null,
      "prefix": null,
      "degree": "MD",
      "source_table": null,
      "assigning_authority": null,
      "name_type_code": null,
      "id_check_digit": null,
      "check_digit_scheme": null,
      "id_type_code": "XX"
    },
    "patient_type": "IN",
    "visit_num": null,
    "financial_class": "15",
    "charge_price_indicator": null,
    "courtesy_code": null,
    "credit_rating": null,
    "contract_code": null,
    "contract_effective_date": null,
    "contract_amt": null,
    "contract_period": null,
    "interest_code": null,
    "transfer_to_bad_debt_code": null,
    "transfer_to_bad_debt_date": null,
    "bad_debt_agency_code": null,
    "bad_debt_transfer_amt": null,
    "bad_debt_recovery_amt": null,
    "delete_account_indicator": null,
    "delete_account_date": null,
    "discharge_disposition": null,
    "discharged_to_location": null,
    "diet_type": null,
    "servicing_facility": "COCBR",
    "reason_for_visit": null,
    "account_status": "ADM",
    "pending_location": null,
    "prior_temporary_location": null,
    "admit_date_time": "201705152228",
    "discharge_date_time": null,
    "current_patient_balance": null,
    "total_charges": null,
    "total_adjustments": null,
    "total_payments": null,
    "alternate_visit_id": null,
    "visit_indicator": null,
    "consulting_doctor": {
      "person_id": ".SELF",
      "family_name": "Referred",
      "given_name": "Self",
      "second_and_further_given_names_or_initials_thereof": null,
      "suffix": null,
      "prefix": null,
      "degree": null,
      "source_table": null,
      "assigning_authority": null,
      "name_type_code": null,
      "id_check_digit": null,
      "check_digit_scheme": null,
      "id_type_code": "XX"
    }
  },
  "0004.ORC": {
    "order_control": "NW",
    "placer_order_num": null,
    "filler_order_num": null,
    "placer_group_num": null,
    "order_status": null,
    "response_flag": "N",
    "quantity_timing": null,
    "parent_order": null,
    "date_time_of_transaction": "201705152359",
    "entered_by": "HSK1",
    "verified_by": null,
    "ordering_provider": null,
    "enterer_location": null,
    "call_back_phone_num": null,
    "order_effective_date_time": null,
    "order_control_code_reason": null,
    "entering_organization": null,
    "entering_device": "BRHDME1W011B.2"
  },
  "0005.OBR": {
    "set_id": "1",
    "placer_order_num": "20170515-0365",
    "filler_order_num": null,
    "universal_service_id": {
      "id": "CONS",
      "text": "PHA",
      "name_of_coding_system": "PHARMACY CONSULT"
    },
    "priority": null,
    "requested_date_time": "201705152359",
    "obsv_date_time_num": null,
    "obsv_end_date_time_num": null,
    "collection_volume": null,
    "collector_id": null,
    "specimen_action_code": null,
    "danger_code": {
      "id": "ACOEISO",
      "text": "Precautions:"
    },
    "relevant_clinical_information": null,
    "specimen_received_date_time": null,
    "specimen_source": null,
    "ordering_provider": {
      "person_id": "SHISAN",
      "family_name": "Kagathur",
      "given_name": "Santosh",
      "second_and_further_given_names_or_initials_thereof": "PGY1",
      "suffix": null,
      "prefix": null,
      "degree": "MD"
    },
    "order_callback_phone_num": null,
    "placer_field_1": null,
    "placer_field_2": null,
    "filler_field_1": null,
    "filler_field_2": null,
    "results_rpt_status_chng_date_time": null,
    "charge_to_practice": null,
    "diagnostic_service_section_id": null,
    "result_status": null,
    "parent_result": null,
    "quantity_timing": {
      "quantity": null,
      "interval": null,
      "duration": null,
      "start_date_time": null,
      "end_date_time": null,
      "priority": "S"
    }
  },
  "0006.NTE": {
    "set_id": "1",
    "source_of_comment": "P",
    "comment": {
      "query_code": "BRPHACON",
      "query_text": "Select Pharmacy Consult",
      "query_element_code": "V",
      "query_response": "Vancomycin Rx to Dose",
      "query_response_name": "Pharmacy Consults Brandon"
    }
  },
  "0007.NTE": {
    "set_id": "2",
    "source_of_comment": "P",
    "comment": {
      "query_code": "HOECAMP",
      "query_text": "Test to be performed at?",
      "query_element_code": null,
      "query_response": "BR"
    }
  },
  "0008.ZMN": {
    "icd_diagnosis": "ZMN"
  }
}